sample-compose/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="co.nimblehq.sample.compose">
+
+    <!--For Chucker in debug build only-->
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
@luongvo Just curious, why don't we put this inside the main manifest? It looks like the generated file 🤔

Reply:
@doannimble We only need this permission for Chucker to work on the debug build. If the app needs to post notifications in the system tray, we will move this to the main Manifest.

Reply:
@luongvo Ah I see, thanks for your clarification 🙏

Reply:
If the main Manifest has 2 permission definitions, the debug Manifest has 1, then the final debug build has 3 while the release build has only 2.